tests/e2e/ctl_v3_migrate_test.go

```diff
@@ -85,11 +83,22 @@ func TestCtlV3Migrate(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	rev := resp.Header.Revision
```

Comment:
rename to `revAfterMigrate` for better readability?

Reply:
Done